frontends/web/src/i18n/i18n.js

```diff
@@ -121,6 +121,9 @@ i18n.on('languageChanged', (lng) => {
             const localeLang = nativeLocale.replace('_', '-').split('-')[0];
             match = lngLang === localeLang;
         }
+        if (lng.toLowerCase() === 'c') {
+            lng = 'en';
```

Comment:
I don't think this is needed: the `languageChanged` callback is triggered only from the language picker which includes only the supported translations plus "en". Is it actually possible for `lng` to be C?

Reply:
you are right, fixed